web/src/admin/providers/ldap/LDAPProviderForm.ts

```diff
@@ -86,12 +86,8 @@ export class LDAPProviderFormPage extends WithBrandConfig(BaseProviderForm<LDAPP
                         const groups = await new CoreApi(DEFAULT_CONFIG).coreGroupsList(args);
                         return groups.results;
                     }}
-                    .renderElement=${(group: Group): string => {
-                        return group.name;
-                    }}
-                    .value=${(group: Group | undefined): string | undefined => {
-                        return group?.pk;
-                    }}
+                    .renderElement=${"name"}
+                    .value=${"pk"}
```

Comment:
My demo that the new shorter declaration scheme works.  And it does!

Reply:
While I do agree this is much shorter and concise, we loose the compile-time type checking of typescript and I think this could make it very easy to introduce bugs